src/test/java/org/apache/commons/io/input/TeeReaderTest.java

```diff
@@ -30,6 +34,54 @@
  */
 public class TeeReaderTest  {
 
+    // Cannot use StringReader or CharSequenceReader as super class because these removed the throws clause from close()
```

Comment:
@robtimus 
Same kind of comment as above, maybe classes these can be made as "Reader" versions of the "OutputStream" classes mentioned above for reuse.